front-end/src/components/KeyMeasureDescription.vue

```diff
@@ -2,20 +2,114 @@
   <div class="description-container">
     <p class="description" v-if="measure.htmlDescription" v-html="measure.htmlDescription"></p>
     <p class="description" v-else-if="measure.description">{{measure.description}}</p>
-    <LogoList v-if="measure.id === 'cinquante'" />
+    <div id="appro-graphs" v-if="measure.id === 'qualite-des-produits'">
+      <div>
+        <p>Exemple de ce qui est attendu fin 2022 :</p>
```

Comment:
On commence à plus s'y retrouver, je mettrai bien chaque description ci-dessous dans un component dédié. Et peut être dans un second temps utiliser les slots : https://v3.vuejs.org/guide/component-slots.html#slot-content

Reply:
Où plutôt avec `:is` : https://v3.vuejs.org/guide/component-basics.html#dynamic-components